Error downloading the auth plugin

From today we started seeing errors in our CI with the logs (removed some env vars for privacy):

```
2024-02-06T17:39:57.0746500Z ##[group]Run google-github-actions/setup-gcloud@v1
2024-02-06T17:39:57.0747010Z with:
2024-02-06T17:39:57.0747292Z   skip_install: false
2024-02-06T17:39:57.0747631Z   version: latest
2024-02-06T17:39:57.0747926Z env:
2024-02-06T17:39:57.0748539Z   ...
2024-02-06T17:39:57.0755746Z ##[endgroup]
2024-02-06T17:39:58.4196208Z [command]/usr/bin/tar xz --warning=no-unknown-keyword --overwrite -C /home/runner/work/_temp/f9d13fe3-9da6-4ed4-957c-e547064bad7d -f /home/runner/work/_temp/d4f8c9e3-ecd1-4deb-928e-ee380c92e1f1
2024-02-06T17:40:17.1850661Z Successfully authenticated
2024-02-06T17:40:17.2008172Z ##[group]Run simenandre/setup-gke-gcloud-auth-plugin@v1
2024-02-06T17:40:17.2008714Z env:
2024-02-06T17:40:17.2010138Z   ...
2024-02-06T17:40:17.2017483Z   CLOUDSDK_METRICS_ENVIRONMENT: github-actions-setup-gcloud
2024-02-06T17:40:17.2018084Z   CLOUDSDK_METRICS_ENVIRONMENT_VERSION: 1.1.1
2024-02-06T17:40:17.2018548Z ##[endgroup]
2024-02-06T17:40:17.2407416Z ##[group]Run REPO_URL="https://packages.cloud.google.com/apt"
2024-02-06T17:40:17.2408239Z [36;1mREPO_URL="https://packages.cloud.google.com/apt"[0m
2024-02-06T17:40:17.2409797Z [36;1mecho "deb [signed-by=/etc/apt/trusted.gpg.d/cloud.google.gpg] $REPO_URL cloud-sdk main" | sudo tee -a /etc/apt/sources.list.d/google-cloud-sdk.list >/dev/null[0m
2024-02-06T17:40:17.2411557Z [36;1mcurl -fsSLo - https://packages.cloud.google.com/apt/doc/apt-key.gpg | sudo gpg --dearmor -o /etc/apt/trusted.gpg.d/cloud.google.gpg[0m
2024-02-06T17:40:17.2412541Z [36;1m[0m
2024-02-06T17:40:17.2412850Z [36;1msudo apt-get update -y[0m
2024-02-06T17:40:17.2413472Z [36;1msudo apt-get install -y google-cloud-sdk-gke-gcloud-auth-plugin[0m
2024-02-06T17:40:17.2458035Z shell: /usr/bin/bash --noprofile --norc -e -o pipefail {0}
2024-02-06T17:40:17.2458598Z env:
2024-02-06T17:40:17.2459240Z   ...
2024-02-06T17:40:17.2466440Z   CLOUDSDK_METRICS_ENVIRONMENT: github-actions-setup-gcloud
2024-02-06T17:40:17.2467042Z   CLOUDSDK_METRICS_ENVIRONMENT_VERSION: 1.1.1
2024-02-06T17:40:17.2467499Z ##[endgroup]
2024-02-06T17:40:17.5994128Z Get:1 file:/etc/apt/apt-mirrors.txt Mirrorlist [142 B]
2024-02-06T17:40:17.6615531Z Get:6 https://packages.microsoft.com/ubuntu/22.04/prod jammy InRelease [3631 B]
2024-02-06T17:40:17.7895497Z Get:7 https://packages.microsoft.com/ubuntu/22.04/prod jammy/main arm64 Packages [33.1 kB]
2024-02-06T17:40:17.8036485Z Get:8 https://packages.microsoft.com/ubuntu/22.04/prod jammy/main armhf Packages [11.3 kB]
2024-02-06T17:40:17.8103665Z Get:9 https://packages.microsoft.com/ubuntu/22.04/prod jammy/main amd64 Packages [125 kB]
2024-02-06T17:40:17.8636686Z Get:10 https://packages.cloud.google.com/apt cloud-sdk InRelease [6361 B]
2024-02-06T17:40:17.9715467Z Get:11 https://packages.cloud.google.com/apt cloud-sdk/main amd64 Packages [599 kB]
2024-02-06T17:40:18.0785663Z Hit:12 https://ppa.launchpadcontent.net/ubuntu-toolchain-r/test/ubuntu jammy InRelease
2024-02-06T17:40:18.6488469Z Hit:2 http://azure.archive.ubuntu.com/ubuntu jammy InRelease
2024-02-06T17:40:18.6538282Z Get:3 http://azure.archive.ubuntu.com/ubuntu jammy-updates InRelease [119 kB]
2024-02-06T17:40:18.6558856Z Hit:4 http://azure.archive.ubuntu.com/ubuntu jammy-backports InRelease
2024-02-06T17:40:18.6585197Z Get:5 http://azure.archive.ubuntu.com/ubuntu jammy-security InRelease [110 kB]
2024-02-06T17:40:18.8578091Z Get:13 http://azure.archive.ubuntu.com/ubuntu jammy-updates/main amd64 Packages [1327 kB]
2024-02-06T17:40:18.8730041Z Get:14 http://azure.archive.ubuntu.com/ubuntu jammy-updates/main Translation-en [268 kB]
2024-02-06T17:40:18.8748674Z Get:15 http://azure.archive.ubuntu.com/ubuntu jammy-updates/restricted amd64 Packages [1346 kB]
2024-02-06T17:40:18.8878673Z Get:16 http://azure.archive.ubuntu.com/ubuntu jammy-updates/restricted Translation-en [220 kB]
2024-02-06T17:40:18.8912636Z Get:17 http://azure.archive.ubuntu.com/ubuntu jammy-updates/universe amd64 Packages [1043 kB]
2024-02-06T17:40:19.0891859Z Get:18 http://azure.archive.ubuntu.com/ubuntu jammy-security/main amd64 Packages [1111 kB]
2024-02-06T17:40:19.0959140Z Get:19 http://azure.archive.ubuntu.com/ubuntu jammy-security/main Translation-en [208 kB]
2024-02-06T17:40:19.0989356Z Get:20 http://azure.archive.ubuntu.com/ubuntu jammy-security/restricted amd64 Packages [1317 kB]
2024-02-06T17:40:19.1110602Z Get:21 http://azure.archive.ubuntu.com/ubuntu jammy-security/restricted Translation-en [215 kB]
2024-02-06T17:40:19.1142971Z Get:22 http://azure.archive.ubuntu.com/ubuntu jammy-security/universe amd64 Packages [838 kB]
2024-02-06T17:40:26.7382822Z Fetched 8900 kB in 3s (3330 kB/s)
2024-02-06T17:40:27.6136262Z Reading package lists...
2024-02-06T17:40:27.6800544Z Reading package lists...
2024-02-06T17:40:27.8331228Z Building dependency tree...
2024-02-06T17:40:27.8343986Z Reading state information...
2024-02-06T17:40:27.8853642Z Some packages could not be installed. This may mean that you have
2024-02-06T17:40:27.8854758Z requested an impossible situation or if you are using the unstable
2024-02-06T17:40:27.8855777Z distribution that some required packages have not yet been created
2024-02-06T17:40:27.8856601Z or been moved out of Incoming.
2024-02-06T17:40:27.8858711Z The following information may help to resolve the situation:
2024-02-06T17:40:27.8909219Z 
2024-02-06T17:40:27.8909494Z The following packages have unmet dependencies:
2024-02-06T17:40:27.9662616Z  google-cloud-cli-gke-gcloud-auth-plugin : Breaks: google-cloud-sdk-gke-gcloud-auth-plugin but 463.0.0-0 is to be installed
2024-02-06T17:40:27.9691134Z E: Unable to correct problems, you have held broken packages.
2024-02-06T17:40:27.9723457Z ##[error]Process completed with exit code 100.
```

I think this is probably related to the apt repo, but I am not sure where to report such a thing

--- a/README.md
+++ b/README.md
@@ -70,7 +70,7 @@ Assuming this is your current setup using this plugin:
     location: europe-west1-b
 ```
 
-Migrating to get-gre-credentials can be done like so:
+Migrating to get-gke-credentials can be done like so:
 
 ```yaml
 - name: Authenticate to Google

--- a/README.md
+++ b/README.md
@@ -1,6 +1,6 @@
 # `simenandre/setup-gke-gcloud-auth-plugin`
 
-## ⚠️ This plugin is deprecated. See [Migration Guide](#migration-guide-to-google-github-actionsget-gke-credentialsv2)
+## ⚠️ This plugin is deprecated. See the [migration guide](#migration-guide-to-google-github-actionsget-gke-credentialsv2) for how to achieve the same using Google's own github action
 
 This GitHub Actions installs a pre-requisite for "gcloud container clusters get-credentials"
 command with a modern k8s client.

--- a/README.md
+++ b/README.md
@@ -1,5 +1,7 @@
 # `simenandre/setup-gke-gcloud-auth-plugin`
 
+## ⚠️ This plugin is deprecated. See [Migration Guide](#migration-guide-to-google-github-actionsget-gke-credentialsv2)
+
 This GitHub Actions installs a pre-requisite for "gcloud container clusters get-credentials"
 command with a modern k8s client.
 
@@ -49,4 +51,38 @@ jobs:
 
       # At this point you should be able to communicate with your cluster
       - run: kubectl get pods
+```
+
+### Migration guide to [google-github-actions/get-gke-credentials@v2](https://github.com/google-github-actions/get-gke-credentials)
+
+Assuming this is your current setup using this plugin:
+
+```yaml
+- name: Authenticate to Google
+  uses: "google-github-actions/auth@v1"
+  with:
+    credentials_json: "${{ secrets.GCP_SERVICEACCOUNT_KEY }}"
+- uses: simenandre/setup-gke-gcloud-auth-plugin@v1
+- name: Authenticate to GKE cluster
+  uses: google-github-actions/get-gke-credentials@v1
+  with:
+    cluster_name: my_cluster
+    location: europe-west1-b
+```
+
+Migrating to get-gre-credentials can be done like so:
+
+```yaml
+- name: Authenticate to Google
+  uses: "google-github-actions/auth@v2"
+  with:
+    credentials_json: "${{ secrets.GCP_SERVICEACCOUNT_KEY }}"
+- uses: "google-github-actions/setup-gcloud@v2"
+  with:
+    install_components: "gke-gcloud-auth-plugin"
+- name: Authenticate to GKE cluster
+  uses: google-github-actions/get-gke-credentials@v2
+  with:
+    cluster_name: my_cluster
+    location: europe-west1-b
 ```

--- a/action.yml
+++ b/action.yml
@@ -7,6 +7,10 @@ branding:
 runs:
   using: composite
   steps:
+    - name: Inform about deprecation
+      shell: bash
+      run: |
+        echo "This action is deprecated. See the readme for a migration guide: https://github.com/simenandre/setup-gke-gcloud-auth-plugin"
     - name: Install gke-gcloud-auth-plugin
       shell: bash
       run: |

